spec/requests/feedbacks_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe "Feedbacks", type: :request do
+  describe "POST /contact_us" do
+    it "go to /contact_us, create feedback and redirect to root with notice" do
+      feedback_params = FactoryBot.attributes_for :feedback
```

Comment:
у вас заінклуджені хелпери факторі бота, можна просто так 
```suggestion
      feedback_params = attributes_for :feedback
```